src/main/java/com/dougnoel/sentinel/steps/TextVerificationSteps.java

```diff
@@ -68,10 +68,12 @@ public static void verifyElementTextIsEmpty(String elementName, String assertion
      * @param matchType String whether we are doing an exact match or a partial match
      * @param text String The text to verify exists in the element.
      */
-    @Then("^I verify the (.*?)( does not)? (has|have|contains?) the text \"([^\"]*)\"$")
-    public static void verifyElementTextContains(String elementName, String assertion, String matchType, String text) {
+    @Then("^I verify the (.*?)( tooltip)?( does not)? (has|have|contains?) the text \"([^\"]*)\"$")
+    public static void verifyElementTextContains(String elementName, String assertion, String tooltip, String matchType, String text) {
         boolean negate = !StringUtils.isEmpty(assertion);
+        boolean hasTooltip = !StringUtils.isEmpty(tooltip);
         String negateText = negate ? "not " : "";
+        String hasTooltipText = hasTooltip ? "not " : "";
```

Comment:
Just putting the word tooltip or nothing.
```suggestion
        String hasTooltipText = hasTooltip ? "tooltip " : "";
```